src/components/AuditLogsTable/AuditLogsTableActions/AuditLogsTableActions.test.tsx

```diff
@@ -101,6 +101,5 @@ describe("AuditLogsTableActions", () => {
     await userEvent.click(dropdownMenu);
     await userEvent.selectOptions(dropdownMenu, "100/page");
     expect(dropdownMenu).toHaveTextContent("100/page");
-    expect(window.location.search).toEqual("?limit=100");
```

Comment:
As you mentioned in the PR description, this should check that the `updateAuditEventsLimit` action was dispatched. There's an example of how we check that actions are dispatched here: https://github.com/canonical/juju-dashboard/blob/audit-logs/src/components/AuditLogsTable/AuditLogsTable.test.tsx#L87-L97.